draft/accessibility.md

```diff
@@ -0,0 +1,97 @@
+# Accessibility Team
+
+This charter supersedes [DEP 11: Accessibility Team](https://github.com/django/deps/blob/main/final/0011-accessibility-team.rst). The Accessibility Team encourages projects maintained by the Django Software Foundation to be accessible to as many people as possible, particularly those with disabilities that make using the web more difficult.
+
+## Scope of responsibilities
+
+The remit of the Accessibility Team shall include all user-facing components
+of projects maintained by the Django Software Foundation. This includes but is
+not limited to:
+
+- User-facing parts of Django, such as default HTML output in forms and such,
+  and the Django admin and its default theme, HTML, and UI components.
+- All websites and other software projects maintained by the DSF, such as the
+  Django website, projects' documentation, Django Snippets, the issue tracker, etc.
+
+The responsibilities of the Accessibility Team are expected to change over time, and are to be decided by consensus of the team, with input from the Technical Board as required. To begin, several areas have been identified:
```

Comment:
I dunno if we have any guidelines about line length but I would like to define a max line length. But that's unrelated to this particular change :)